ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.6.yaml

```diff
@@ -22,7 +22,11 @@ tests:
     test:
     - as: test
       cli: latest
-      commands: VERBOSE=2 make -k test-acceptance test-acceptance-artifacts
+      commands: OPERATOR_REPO_REF=$(cat /usr/local/quay.credentials | cut -d "=" -f1) QUAY_USERNAME=$(cat /usr/local/quay.credentials | cut -d "=" -f2) QUAY_TOKEN=$(cat /usr/local/quay.credentials | cut -d "=" -f3) VERBOSE=2 make -k release-operator test-acceptance-with-bundle test-acceptance-artifacts
+      credentials:
+      - mount_path: /usr/local/quay.credentials
+        name: redhat-developer-services-binding-operator-test-quay-credentials
+        namespace: ocpappsvc-secrets
```

Comment:
This should match the destination into which your secret is synced in https://github.com/openshift/release/pull/14748/files